docs/maps/import-geospatial-data.asciidoc

```diff
@@ -0,0 +1,30 @@
+[role="xpack"]
+[[import-geospatial-data]]
+== Import geospatial data
+
+In order to use your geospatial data in the Elastic Stack, it must be indexed as {ref}/geo-point.html[geo_point] or {ref}/geo-shape.html[geo_shape].
+Kibana makes it easy to import geospatial data into the Elastic Stack.
+
+=== Upload a CSV file with latitude and longitude columns
+
+To upload a CSV file with latitude and longitude columns
```

Comment:
This sentence repeats the title.  Is it needed?

Reply:
> This sentence repeats the title. Is it needed?

I will change it to address your question "Are CSV Upload and GeoJSON upload two different ways to get your data into Elasticsearch and have it set up correctly"